### PR TITLE
chore: cleanup recent history entries without PR title

### PR DESCRIPTION
**Pull request by jahorton**
@keymanapp-test-bot skip

### CODE REVIEW

**Review by mcdurdin — APPROVED** [review]

**Comment by mcdurdin** [comment]
Have we identified how this happened?

File: HISTORY.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 ## 17.0.232 alpha 2023-12-18
 
-*  (#10270)
+* feat(web): adds highlighting to BKSP while held (#10270)
```
Comment (jahorton) [inline HISTORY.md:21]: Blank entry added with https://github.com/keymanapp/keyman/pull/10277.

File: HISTORY.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 * fix(mac): Improve detection of non-compliant apps (#10282)
 * fix(web): default multitap logic for shift-layer shift multitaps (#10281)
-*  (#10284)
+* fix(web): keyboard functionality on Android 5.0 / Chrome 35 (#10284)
```
Comment (jahorton) [inline HISTORY.md:11]: Blank entry (and its non-blank siblings) were added with #10288.

**Review by jahorton — COMMENTED** [review]
> Have we identified how this happened?

Not to my knowledge.  Is there a procedure we've used in the past to discern the cause for prior cases?  I tried pulling the release tag for one of the commits and running the history-editor on it, but I'm having trouble making the right kind of token that can pass the `octokit` credential checks needed.

File: HISTORY.md (hunk 1 above)
Comment (mcdurdin) [reply HISTORY.md:21]: https://build.palaso.org/buildConfiguration/Keyman_TriggerReleaseBuildsMaster/431480?buildTab=log&focusLine=238&logView=flowAware&linesState=199

```
01:01:58   # Validating history for 17.0.232
01:01:59   Found commits for ["log","--merges","--first-parent","--format=%H","origin/master","a84e5202d9d9a3b2103a5a09ac868a698fdfb7cf..origin/master"]:
01:01:59   84a839360285468f93da3b2d3e987e4d58c28f2d
01:01:59   -- Adding ## 17.0.232 alpha 2023-12-18
01:01:59   -- Adding *  (#10270)
01:01:59   History has changed; writing updated history
01:02:09   Creating comment on Pull Request #10270
01:02:10   # 1 change(s) found for 17.0.232
01:02:10   
01:02:10   # Incrementing version from 17.0.232
01:02:10   # New version is 17.0.233
```

**Comment by mcdurdin** [comment]
OK, well, it's sporadic and rare so pushed that into a separate issue to investigate in 18.0

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.241-alpha](https://keyman.com/downloads/releases/alpha/17.0.241)

**Pull request merged**